test/integration/commands/pluck_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Integration.Commands.PluckTest do
+  use Cog.AdapterCase, adapter: "test"
+
+  setup do
+    user = user("lucky", first_name: "Shamaus", last_name: "McLucky")
+    |> with_chat_handle_for("test")
+
+    {:ok, %{user: user}}
+  end
+
+  test "with no field option", %{user: user} do
```

Comment:
So, I have this code lying around from some previous work that ended up getting tabled, and I never got around to extracting it. These functions would be good to add to a command test utility module and would help clean up these tests immensely.

```elixir
 defp seed(input, pipeline) when is_list(input) or is_map(input),
    do: "@bot: seed '#{Poison.encode!(input)}' | #{pipeline}"

  # Currently, it's impossible to get a JSON array of data out of the
  # end of a pipeline. Instead, we get newline-separated JSON strings
  # of each individual data map in that array
  #
  # Example:
  #     "{\n  \"foo\": 123\n}\n{\n  \"foo\": 321\n}"
  #
  # This function recovers an Elixir data structure from this string,
  # by relying on the fact that the boundary between maps will be
  # "}\n{", which is done with a regex with positive lookahead and
  # lookbehind.
  #
  # Now you have `n` problems.
  defp unmangle_multiple_output(output_string),
    do: output_string |> String.split(~r/(?<=})\n(?={)/) |> Enum.map(&Poison.decode!(&1, keys: :atoms!))

```